pkg/network/common/networkvalidation/network.go

```diff
@@ -0,0 +1,146 @@
+package networkvalidation
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/library-go/pkg/config/validation"
+	"github.com/openshift/library-go/pkg/crypto"
+	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
+)
+
+func ValidateInClusterNetworkNodeConfig(config *configapi.NodeConfig, fldPath *field.Path) validation.ValidationResults {
```

Comment:
@deads2k this is new or where does this came from?

Reply:
it was originally a move that needed some package updates.